lib/cc/yaml/nodes/glob.rb

```diff
@@ -2,8 +2,11 @@ module CC
   module Yaml
     module Nodes
       class Glob < Scalar
-        def match?(path)
-          File.fnmatch(to_s, path)
+        DOUBLE_STAR_END = Regexp.new('\/\*\*$').freeze
+        DOUBLE_STAR_BEGIN = Regexp.new('\*\*\.').freeze
```

Comment:
I don't see a ton of value in these constants. Personally, I'd inline them.

:lipstick: our style guide says `%r{ }` for regexes with slashes in it.